src/editor/EditorStartup.js

```diff
@@ -314,6 +314,16 @@ class EditorStartup {
       return true
     })
 
+    // adding a mousedown listener to handle some dropdowns closing
```

Comment:
could this mousedown event be trapped inside the selist component that would close itself?
The idea is to avoid spreading logic in multiple place.